packages/block-library/src/heading/edit.js

```diff
@@ -38,12 +38,14 @@ function HeadingEdit( { attributes, setAttributes, mergeBlocks, onReplace } ) {
 						setAttributes( { level: newLevel } )
 					}
 				/>
-				<AlignmentToolbar
-					value={ align }
-					onChange={ ( nextAlign ) => {
-						setAttributes( { align: nextAlign } );
-					} }
-				/>
+				{ Platform.OS !== 'android' && (
```

Comment:
This isn't working for me. I think because this is our re-exported `Platform` and not the react-native `Platform`. I'm seeing  `Platform.OS` with a value of `native`. Do we have to use `Platform.select` here?